tests/foreman/cli/test_satellitesync.py

```diff
@@ -230,20 +433,19 @@ def class_export_entities():
 
 
 @pytest.fixture(scope='function')
-def function_export_cv_directory():
-    """Create directory for CV export and at the end remove it"""
-    export_dir = f'{get_export_base()}/{gen_string("alpha")}'
-    ssh.command(f'mkdir {export_dir}')
+def export_cv_dir(module_org):
+    """Get directory for CV export and at the end remove it"""
+    export_dir = '/var/lib/pulp/exports/'
```

Comment:
Why not ensure the directory exists via mkdir, as was done previously?

Reply:
Previously, you have to create some directory and put the exports in.  New implementation puts export files in `/var/lib/pulp/exports/` by default.